app/src/sharedTest/java/org/oppia/android/app/home/promotedlist/PromotedStoryViewModelTest.kt

```diff
@@ -91,9 +91,9 @@ class PromotedStoryViewModelTest {
   fun testPromotedStoryViewModelEquals_reflexiveBasicPromotedStoryViewModel_isEqual() {
     launch<HomeFragmentTestActivity>(
       HomeFragmentTestActivity.createHomeFragmentTestActivity(context)
-    ).use {
-      it.onActivity {
-        val promotedStoryViewModel = createBasicPromotedStoryViewModel(it)
+    ).use { homeFragmentTestActivity ->
+      homeFragmentTestActivity.onActivity { homeFragmentTestActivity ->
```

Comment:
You should not name both the lambda functions same.

For above use `homeFragmentTestActivityScenario` and for internal use `homeFragmentTestActivity`

Reply:
Ditto else where.

Something like this:
```
    ).use { homeFragmentTestActivityScenario ->
      homeFragmentTestActivityScenario.onActivity { homeFragmentTestActivity ->
        val promotedStoryViewModel = createBasicPromotedStoryViewModel(homeFragmentTestActivity)
```

Reply:
@jonathanalvares9009 can you please reply to this.

Reply:
@rt4914 done that and sent a pull request